### ISSUE TITLE
Use favicons for standard links instead of default linkace icon

### ISSUE THREAD

**Issue by cantunborn** [Kovah/LinkAce#302]
**Is your feature request related to a problem? Please describe.**
Default icon is used for standard links, when favicons could be used.

**Describe the solution you'd like**
Fetch favicons (if present) from sites, and use linkace's default icon as fallback (currently only this icon is used for standard links).

**Describe alternatives you've considered**

**Additional context**
Favicons make it easier to identify stuff and using the same icon for links makes them harder to tell apart. Even though tags can be used to tell links apart, images are easier to remember.

**Comment by webysther** [comment]
@Kovah any change to add ability to choose a image (png/ico) for link/list/tag? I use the bookmark without text on chrome to create a "fast panel" and with this I can migrate all my bookmarks and create also groups using lists or tags:

![image](https://github.com/Kovah/LinkAce/assets/750007/d26b31e1-c3ce-4499-ad22-ae9a7f0a0859)

**Comment by webysther** [comment]
A sample: https://webcrate.app/docs/crates

![image](https://github.com/Kovah/LinkAce/assets/750007/300dde95-258e-4832-b949-9e5339ad3b34)

**Comment by Kovah** [comment]
I will have a look if this is easy to implement. If not, I have to push it into the future.

**Comment by webysther** [comment]
Thanks a lot @Kovah, do you use HomeAssistant? The ideia of using mdi:* icons maybe work as well, this can be usefull for list/tags.

**Comment by Kovah** [comment]
Yes I'm using Home Assistant. What do you mean with mdi icons?

**Comment by webysther** [comment]
> Yes I'm using Home Assistant. What do you mean with mdi icons?

![image](https://github.com/Kovah/LinkAce/assets/750007/ba649bd2-6797-42d8-9879-71334247c7f5)

[Home assistant use icons](https://www.home-assistant.io/docs/frontend/icons/) and maybe can be a alternative for custom icon in list/tags/etc

**Comment by Kovah** [comment]
Ah alright! That would probably mean to include a whole set of icons instead of having dedicated ones. Will keep this as an idea for v2.

**Comment by webysther** [comment]
> Ah alright! That would probably mean to include a whole set of icons instead of having dedicated ones. Will keep this as an idea for v2.

Yes, a custom icon is much better for my personal point of view, but another way is using a set of icons like home assistant do, for list/tags the set of icons sounds a better choice. For private domain for instance a custom bookmarks maybe works better.
